canonical/assets.ubuntu.com
# Add deperacted option to assets

## Done

- Add deprecated option to assets: this is an MVP, later can be UX and UI reviewed

## QA

- Check out this feature branch
- Run the site using the command `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8017
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click on the Edit button, and check an asset as deprecated
- Go back to the assets list and make sure that the deprecated badge is shown
- Make sure that the deprecated field is shown on the API side `/v1?q=query`
- Edit another asset using the API: `PUT /v1/url-to-asset` with body `deprecated=true`
- Make sure that the field value is applied

## Issue / Card

Fixes #98

## Screenshots


![0 0 0 0_8017_manager_q=+ type= (1)](https://user-images.githubusercontent.com/36013798/231557064-46b266f2-0d0a-4cf2-ab0a-8b3008f9eefb.png)
